website/static/schemas/next/searchCriteria.schema.json

```
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/searchCriteria.schema.json",
+  "type": "object",
+  "title": "SearchCriteria",
+  "allOf": [{ "$ref": "context.schema.json#" }],
+  "properties": {
+    "type": { "const": "fdc3.chat.searchCriteria" },
+    "criteria": {
+      "type": "array",
+      "items": { 
+        "anyOf": [
+          { "$ref":"instrument.schema.json#" },
+          { "$ref": "organization.schema.json#" },
+          { "$ref": "contact.schema.json#" },
+          { "type": "string" }
+        ]
+      }
+    }
+  },
+  "required": ["type", "criteria"]
```

```suggestion
  "required": ["criteria"]
```
Already required by context schema